UnityGLTF/Assets/UnityGLTF/Scripts/Extensions/GLTFSceneExporter.Animation.cs

```diff
@@ -0,0 +1,335 @@
+﻿using System;
+using System.Collections.Generic;
+using GLTF.Schema;
+using UnityEngine;
+using UnityEditor.Animations;
+using UnityEditor;
+using System.Linq;
+
+namespace UnityGLTF
+{
+	public partial class GLTFSceneExporter
+	{
+		private enum Property
+		{
+			m_LocalPosition = GLTFAnimationChannelPath.translation,
```

Comment:
m_ is a more of a C++ paradigm. Just use _localPosition;

Reply:
Also weird for enums. Do:
LocalPosition,
LocalRotation,
LocalScale,
BlendShape